test/nonregression/test_run_iotools.py

```diff
@@ -14,16 +14,41 @@
 __email__ = "arnaud.marcoux@inria.fr"
 __status__ = "Development"
 
-
 import warnings
 import sys
+from testing_tools import create_list_hashes, validate_folder
 from testing_tools import clean_folder, compare_folders
 from testing_tools import identical_subject_list, same_missing_modality_tsv
 from os import pardir
 
 # Determine location for working_directory
 warnings.filterwarnings("ignore")
 
+
+def test_run_Nifd2Bids(cmdopt):
+    from clinica.iotools.converters.nifd_to_bids.nifd_to_bids import convert_clinical_data, convert_images
+    from os.path import dirname, join, abspath
+    import os
+
+    root = join(dirname(abspath(__file__)), pardir, 'data', 'Nifd2Bids')
+
+    clean_folder(join(root, 'out', 'bids'), recreate=True)
+
+    # Data location
+    dataset_directory = join(root, 'in', 'unorganized')
+    bids_directory = join(root, 'out', 'bids')
+    clinical_data_directory = join(root, 'in', 'clinical_data')
+
+    # Conversion
+    to_convert = convert_images(dataset_directory, bids_directory, clinical_data_directory)
+    convert_clinical_data(bids_directory, clinical_data_directory, to_convert)
+
+    validate_folder(bids_directory, join(root, 'ref', 'hashes_nifd.p'))
+
+    clean_folder(join(root, 'out', 'bids'), recreate=True)
+    os.remove(join(root, 'in', 'clinical_data', 'clinical_info.tsv'))
```

Comment:
On a pris la convention que ce qui est dans `in` est immuable. Si nécessaire, copie `clinical_data`dans `out` et lance ton pipeline avec ce nouveau dossier de clinical data. Si besoin (pour t'inspirer), c'est fait notamment dans `test_run_T1VolumeCreateDartel` pour le dossier `join(root, 'out', 'caps')`